python/ray/train/v2/_internal/execution/controller/controller.py

```diff
@@ -283,16 +284,22 @@ def _restart_worker_group(
         )
         placement_strategy = self._scaling_policy.scaling_config.placement_strategy
 
+        worker_group_context = WorkerGroupContext(
```

Comment:
Do you need to add run attempt ID as part of this context?

Reply:
I opted to not put it in yet since we don't do anything with it yet. I will add it when it's used (probably in the upcoming state management PR)